totalRP3_Extended/script/script_operands.lua

```diff
@@ -376,6 +376,23 @@ local OPERANDS = {
 			["GetCameraZoom"] = "GetCameraZoom",
 		},
 	},
+	
+	["char_achievement"] = {
+		numeric = true,
+		codeReplacement = function(args)
+			local completedByIndex = 4;
+			if args[1] == "account" then
+				completedByIndex = 4; -- We get the "completed" return
+			elseif args[1] == "character" then
+				completedByIndex = 13; -- We get the "wasEarnedByMe" return
+			end
+			local id = args[2] or "";
+			return ("({GetAchievementInfo(%s)})[%s]"):format(id, completedByIndex);
```

Comment:
You can encap all the ({GetAchievementInfo(%s)})[%s] in a tostring() to force string comparison on the boolean value.